src/redux/greetings/greetingsSlice.js

```diff
@@ -1,28 +1,28 @@
-import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 
-const ApiData = "http://localhost:3000/random_greeting";
+const ApiData = 'http://localhost:3000/random_greeting';
 
 export const getMessage = createAsyncThunk(
```

Comment:
* Kindly ensure that your Frontend renders random greetings as currently, the API works fine but the frontend doesn't render the expected results
![image](https://user-images.githubusercontent.com/52497246/211143972-9da5c909-278f-4632-aa19-ba1531fcd6b8.png)
![image](https://user-images.githubusercontent.com/52497246/211143977-5741a049-1dfb-4480-ba14-b57b18ba482f.png)